CONTRIBUTING.md

```diff
@@ -36,3 +36,9 @@
 2. Create and push a git tag with a matching format to trigger the release CI.
 
     The git tag format is: `service-vX.Y.Z`, where "X.Y.Z" is replaced with the version number.
+
+## CI
+
+Note that CI checks for building `tauri-plugin-holochain-service` and `tauri-plugin-holochain-service-client` do *not* publish the client & service libraries to a local maven repository, but instead use the versions published to Maven Central.
```

Comment:
```suggestion
Note that the CI jobs for building `tauri-plugin-holochain-service` and `tauri-plugin-holochain-service-client` do *not* publish the client & service libraries to a local maven repository, but instead use the versions published to Maven Central.
```

Reply:
Tried to make it a bit clearer -- the main point is the sentence after: https://github.com/holochain/android-service-runtime/pull/122/commits/11771f7d42080ac53b8aaa2ce3eeab430dce56e4

Reply:
:+1: